src/Calculator/Package.appxmanifest

```diff
@@ -8,7 +8,7 @@
         <Logo>Assets\CalculatorStoreLogo.png</Logo>
     </Properties>
     <Dependencies>
-        <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.17133.0" MaxVersionTested="10.0.22000.0" />
+        <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.18308.0" MaxVersionTested="10.0.22000.0" />
```

Comment:
nit: Let's just make this "18362" since that was a released version of Windows. I don't think anyone is going to test with insider builds earlier than 18362.